### PR TITLE
is not to !=

### PR DESCRIPTION
**Pull request by flynneva**

### CODE REVIEW

File: rosapi/src/rosapi/params.py [hunk 1]
```diff
@@ -221,7 +221,7 @@ def _get_param_names(node_name):
     rclpy.spin_until_future_complete(_node, future)
     response = future.result()
 
-    if response is not None:
+    if (response != None):
```
Comment (mvollrath) [inline rosapi/src/rosapi/params.py]: What is the purpose of this change?  Asking because using `is` to compare to `None` is correct.

**Review by mvollrath — CHANGES_REQUESTED** [review]
Please remove extraneous parenthesis from these conditions.

File: rosapi/src/rosapi/params.py (hunk 1 above)
Comment (flynneva) [reply rosapi/src/rosapi/params.py]: when I compile it on ubuntu 20.04 with foxy i get a stderror while compiling about this line.

```
[Processing: rosapi]                                    
--- stderr: rosapi                                       
/home/flynn/code/ros/foxy/install/rosapi/lib/python3.8/site-packages/rosapi/params.py:122: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if default is not "":
---
Finished <<< rosapi [49.6s]
```

Comment (flynneva) [reply rosapi/src/rosapi/params.py]: ill remove the parenthesis though

Comment (flynneva) [reply rosapi/src/rosapi/params.py]: that error is only for line 122, so im not sure if the second "is not" needs to be changed

Comment (mvollrath) [reply rosapi/src/rosapi/params.py]: Comparing to empty string value isn't the same as comparing to `None` which is a singleton.

Comment (flynneva) [reply rosapi/src/rosapi/params.py]: just trying to help out here...

**Review by mvollrath — APPROVED** [review]

**Comment by mvollrath** [comment]
Thanks for the fix, should I be concerned about CI failures?

**Comment by flynneva** [comment]
> Thanks for the fix, should I be concerned about CI failures?

I mean....it looks like it doesnt even get to building and testing your package for the ones that failed because it fails before for other packages (rosidl, i think). they might not be supported for those distros/OS?

Also though the ros ci action isnt that robust yet so maybe its an issue with the action itself rather than an actual build issue? I think i tried this package on eloquent and dashing before on linux and it worked ok (small setup issues, but solvable).

**Comment by flynneva** [comment]
also i just realized i think the build errors might be what the other ROS2 Eloquent PR is for, right?
